x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -807,7 +815,22 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     }
 
     tlog(this.logger, `Timeline alerts to return: ${alertsToReturn.length}`);
-    return alertsToReturn;
+    return alertsToReturn || [];
+  }
+
+  public async fetchTimelineEndpointAlerts(rangeFrom: string, rangeTo: string) {
```

Comment:
Can we make `fetchTimelineAlerts` public and just call that instead of having 2 wrapper methods? If the concern is about alertsIndex being undefined we could include a check where `pseudocode: index != falsy` on [line 709](https://github.com/elastic/kibana/blob/11c8e9ad6ba012eb605cc106e81203914d51ca09/x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts#L709) or even let it fail through since i'm assuming `esClient.openPointInTime` would fail for a falsy index

Reply:
done